kernel/init.c

```diff
@@ -422,14 +425,15 @@ FUNC_NORETURN void _Cstart(void)
 	 */
 	char dummy_thread_memory[sizeof(struct k_thread)];
```

Comment:
Might be slightly more efficient to declare this as `char dummy_thread_memory[...] = {0}` instead of calling `memset()`.